assets/js/script.js

```diff
@@ -0,0 +1,291 @@
+//Part 1 - Shuffle array, add a random number btw 1-4 into array(for adding rounds)
+var instructorMoves = [37,38,39,40]
+var whichPlayer = 0
+var playerTest = 0
+var scoreP1 = 0
+var scoreP2 = 0
+var counterP1 = 0
+var counterP2 = 0
+var rounds = 1
+
+
+function shuffleArray(arr) {
+  for (var i = arr.length - 1; i > 0; i--){
```

Comment:
Explain this line of codes to me, why does the loop reversing? and how does the es6 code works?

Use simple example.